libs/application/templates/passport/src/forms/Draft.ts

```diff
@@ -162,26 +151,9 @@ export const Draft: Form = buildForm({
               title: m.dropLocation,
               titleVariant: 'h3',
               space: 2,
-              description: m.dropLocationDescription,
+              description: 'Þjóðskrá',
```

Comment:
This is untranslate-able unless tied to a message. Perhaps make a new message or update the old one in both locales in contentful?